source/docs/articles/drupal/8/moving-out-of-testing.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Moving Drupal 8 Sites Out of Testing
+description: Pantheon sites created during the Drupal 8 testing period will be deleted at some point in the future. Follow these instructions to keep your site on the platform.
+---
+The Drupal 8 testing organization owns all Drupal 8 sites created on the platform prior to October 9, 2015. All of these sites will be deleted on <DATE>. To keep your site running on the platform, follow these instructions:
```

Comment:
@bmackinney do we have a date to include here? Suggest we include it here before adding the doc to the MVP PR.